olp-cpp-sdk-core/include/olp/core/thread/TaskSchedulerFactory.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <olp/core/CoreApi.h>
+#include <olp/core/porting/make_unique.h>
+
+#include "ThreadPoolTaskScheduler.h"
+
+namespace olp {
+namespace thread {
+class TaskScheduler;
```

Comment:
Already included by ThreadPoolTaskScheduler.h

Also, I would include in header files via <>, even for local headers, as they will be installed later in system folder and it might get messi. In cpp the inclusion should be via "" preferably.

Reply:
done